sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1197,6 +1199,7 @@ ansiNonReserved
 // The non-reserved keywords are listed in `nonReserved`.
 // These 2 together contain all the keywords.
 strictNonReserved
+//--ANSI-STRICT-NON-RESERVED-START
```

Comment:
this is not ansi, we should just call it `STRICT-NON-RESERVED`

Reply:
oh, I missed that. I'll update soon.